test/expect/TestJit.test_backward_opaque.expect

```diff
@@ -1,10 +1,10 @@
-graph(%1 : Double(3, 3)
-      %2 : Double(3, 3)
+graph(%0 : Double(3, 3)
+      %1 : Double(3, 3)
       -------- stage 1 --------
-      %4 : Double(3, 3)) {
-  %3 : Double(3, 3) = cross[dim=-1](%1, %2), uses = [%0.i0];
+      %3 : Double(3, 3)) {
+  %2 : Double(3, 3) = cross[dim=-1](%0, %1)
   ---------------- stage 1 ----------------
-  %5 : Double(3, 3) = cross[dim=-1](%2, %4), uses = [%0.i1];
-  %6 : Double(3, 3) = cross[dim=-1](%4, %1), uses = [%0.i2];
-  return (%3, %5, %6);
+  %4 : Double(3, 3) = cross[dim=-1](%1, %3)
+  %5 : Double(3, 3) = cross[dim=-1](%3, %0)
+  return (%2, %4, %5);
```

Comment:
Haha that's not really opaque anymore

Reply:
lol, the changes to remove opaque stuff from the autograd have been working, but they have also been messing up our tests...